contracts/orderbook/Enumerates/EnumOrders.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.8.4;
```

Comment:
you have two identical libraries, you better have a library that implements a data structure that you like .e.g EnumerableUintSet ... 

and then e.g `using EnumerableUint256Set for EnumerableUint256Set.Uint256Set;`

Reply:
hmm. my bad they are not identical. struct is different

Reply:
still name them as data structure, and then use